notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    TF = "2.5".replace(".", "-")
</pre><p>question: Won't <code>TF[0]</code> always evaluate to "2"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
updated